common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -94,6 +94,7 @@
                     new_time: time,
                     type: type
                 });
+                this.emitPlayVideoEvent = true;
```

Comment:
I do not understand two things why in onPlay: function() this.emitPlayVideoEvent = false;. Do we want to emit event onPlay, will setting onPlay function this.emitPlayVideoEvent = true resolve this issue ?. Why we are setting  this.emitPlayVideoEvent = false; in onPlay ? does setting it to true generates some extra events which we do not need ?.